test/sanity/code-smell/type-boilerplate.py

```diff
@@ -0,0 +1,1004 @@
+#!/usr/bin/env python
```

Comment:
This would be better named `metaclass-boilerplate` instead of `type-boilerplate`.

Reply:
Done